geode-redis/README.md

```diff
@@ -159,53 +159,53 @@ start server \
 |-----------------------|-------------------------------------------------------|-----------------------------------|
 | APPEND             	| BITCOUNT                                           	| ACL CAT                       	|
 | AUTH               	| BITOP                                              	| ACL DELUSER                   	|
-| DEL                	| BITPOS                                             	| ACL GENPASS                   	|
-| EXISTS             	| DBSIZE                                             	| ACL GETUSER                   	|
-| EXPIRE             	| DECR                                               	| ACL HELP                      	|
```

Comment:
Just noticed DECR is in the unsupported column here. I guess that could be changed in a different PR though.

Reply:
we added a ticket to the backlog for double checking that the docs are correct with the state of the support/1.14 branch